css-typed-om/Overview.bs

```diff
@@ -2845,11 +2845,33 @@ of serializations of the contained {{CSSTransformComponent}} objects.
 {{CSSURLImageValue}} objects are serialized to the string given by
 "url(" + {{CSSURLImageValue/url}} + ")".
 
-{{CSSFontFaceValue}} Serialization {#fontfacevalue-serialization}
```

Comment:
Does this actually remove the FontFace serialization?

It's fine, we probably want to get rid of it, just strange to do both at once.